packages/browser/src/gen/index.d.ts

```diff
@@ -13209,7 +13209,7 @@ export namespace Browser {
             /**
              * A list of request types. Requests that cannot match any of the types will be filtered out.
              */
-            types?: ResourceType[] | undefined;
+            types?: (ResourceType | `${ResourceType}`)[] | undefined;
```

Comment:
Downside to this approach. Here, `ResourceType` is actually a string literal union. There's another enum further down that is named `ResourceType`, so this type was also modified.